src/components/footer/getLocationCookie.ts

```diff
@@ -1,10 +1,13 @@
+import type { CountryCode } from '@guardian/libs';
+
 let countryCode: string | null = null;
```

Comment:
I realise that this wasn't added in this PR, but do we know why we're mutating `countryCode` here? It feels like `getCookie` is a fairly inexpensive operation, so it shouldn't matter too much if we end up re-fetching it at some point, and the dependency on state here seems likely to make the logic harder to reason about.

Reply:
Yeah I don't think we need this variable here

Reply:
> One small question: is the mobile asset meant to be different? The G is in front of the white hands, whereas on desktop it’s behind…
> 

Keen eye! Yes this is also to be updated with another image shortly